expr.go

```diff
@@ -1993,6 +1993,28 @@ func evalExpr(e *expr, from, until int32, values map[metricRequest][]*metricData
 			return percentile(values, percent, interpolate)
 		})
 
+	case "removeBelowValue":
+		args, err := getSeriesArg(e.args[0], from, until, values)
+		if err != nil {
+			return nil
+		}
+		threshold, err := getFloatArg(e, 1)
+		if err != nil {
+			return nil
+		}
+
+		var results []*metricData
+		for _, metric := range args {
+			for i, value := range metric.Values {
+				if value < threshold {
+					metric.Values[i] = math.NaN()
```

Comment:
You'll want to wrap this in a forEachSeriesDo block since it's a mapping function for one or more series.